packages/application-to-register-webapp/src/routes/land/check-legal-agreement-details.js

```diff
@@ -1,15 +1,19 @@
 import constants from '../../utils/constants.js'
 import path from 'path'
 import moment from 'moment'
+import { getRegistrationTasks } from '../../utils/helpers.js'
 
 const handlers = {
   get: async (request, h) => {
     const context = await getContext(request)
     return h.view(constants.views.LEGAL_AGREEMENT_SUMMARY, context)
   },
   post: async (request, h) => {
+    const registrationTasks = getRegistrationTasks(request)
+    registrationTasks.taskList.find(task => task.taskTitle === 'Legal information').tasks[0].status = 'COMPLETED'
+    request.yar.set(constants.redisKeys.REGISTRATION_TASK_DETAILS, registrationTasks)
     const context = await getContext(request)
-    return h.redirect(`${constants.views.LEGAL_AGREEMENT_SUMMARY}`, context)
+    return h.redirect(`/${constants.views.REGISTER_LAND_TASK_LIST}`, context)
```

Comment:
Please use the `constants.routes.REGISTER_LAND_TASK_LIST` constant as this already has the prepended slash.

Also remove the context object, it is unnecessary for a redirect.

Reply:
Removed the context object. But Left the constant as it is the same in master branch

Reply:
Updated